src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -77,7 +97,7 @@ internal SarifErrorListItem()
             this.Properties = new ObservableCollection<KeyValuePair<string, string>>();
         }
 
-        public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePath, ProjectNameCache projectNameCache)
+        public SarifErrorListItem(CodeAnalysis.Sarif.Run run, int runIndex, Result result, string logFilePath, ProjectNameCache projectNameCache)
```

Comment:
>CodeAnalysis.Sarif. [](http://example.com/codeflow?start=34&length=19)

I assume this is because one of the new references has a type called Run. I'd prefer using Run = Microsoft.CodeAnalysis.Sarif.Run to avoid using the namespace here.